Source/WTF/wtf/text/WTFString.cpp

```diff
@@ -690,6 +691,42 @@ CString String::latin1() const
 }
 
 #if OS(MORPHOS) || OS(AMIGAOS)
+#if OS(AMIGAOS)
+ULONG GetLength(APTR str, LONG bytes, ULONG mib)
```

Comment:
When reading the MorphOS SDK, the function returns LONG, not ULONG... 

Btw, the same Morphos SDK guide contains a spelling error in the SYNOPSIS: GetLenght instead of GenLength.